utils/src/lib.rs

```diff
@@ -289,14 +289,22 @@ fn get_binary_in<P: AsRef<Path>>(dir: P, name: &str) -> io::Result<path::PathBuf
     Ok(dir.as_ref().join(format!("{}.exe", name)))
 }
 
-#[cfg(unix)]
+#[cfg(all(unix, not(target_os = "macos")))]
 fn get_binary_in<P: AsRef<Path>>(dir: P, name: &str) -> io::Result<path::PathBuf> {
     use std::os::unix::fs::PermissionsExt;
     let bin = dir.as_ref().join(name);
     fs::set_permissions(bin.as_path(), fs::Permissions::from_mode(0o755))?;
     Ok(bin)
 }
 
+#[cfg(all(unix, target_os = "macos"))]
```

Comment:
I have noticed that the shipped bsdiff/bspatch were dynamically linked aarch64 executables. 
Maybe a `target_arch = "aarch64"` condition should be added here, to indicate that the legacy x86_64 support for Mac is missing at now.

Reply:
May have been my omission,  I thought they were universal binaries.  I'll look into this to rectify the situation.

Reply:
Github actions for running test has been added on the master branch, where `os: macos-latest` is temporarily removed because github actions seems to be [lack of arm64 containers support](https://github.com/actions/runner-images/issues/2552#issuecomment-771478000). It might be helpful.